doc/page01_install.md

```diff
@@ -14,25 +14,25 @@ Installation & Quick Start {#install}
 $ git clone -b main https://github.com/jfalcou/eve.git
 @endverbatim
 
-This retrieves the `main` branch which contains the latest stable version. 
+This retrieves the `main` branch which contains the latest stable version.
```

Comment:
We don't update the main branch. Let's just use tags and that's it?

Reply:
We do. The tags points to commit on the main branch.